### DISCUSSION TITLE
Upwork ID verify

### DISCUSSION THREAD

**Discussion by doyen001** [Huniko-Team/community#60] [General]
Hello everybody.
Do you know how to get ID verification badges on the Upwork website?

**Comment by AringoldX** [comment]
To get an ID verification badge on Upwork, you generally need to go through a verification process to confirm your identity. 
Upwork may offer different types of verification options. For example, they might have an ID verification process where you upload a scan or photo of your government-issued ID. Other verification methods could include video verification calls or address verification. Select the appropriate option based on your preference and availability.
